src/chi-vue/src/components/data-table-bulk-actions/DataTableBulkActions.tsx

```diff
@@ -1,4 +1,4 @@
-import { Component, Prop, Vue } from 'vue-property-decorator';
+import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
 import { findComponent } from '@/utils/utils';
```

Comment:
i was playing around in the complex example and i think i found some bugs / it's not working correctly :S